models/langchain4j-community-zhipu-ai/src/main/java/dev/langchain4j/community/model/zhipu/ZhipuAssistantChatModel.java

```diff
@@ -0,0 +1,167 @@
+package dev.langchain4j.community.model.zhipu;
+
+import static dev.langchain4j.internal.Utils.getOrDefault;
+import static dev.langchain4j.spi.ServiceHelper.loadFactories;
+
+import dev.langchain4j.community.model.zhipu.assistant.AssistantKeyValuePair;
+import dev.langchain4j.community.model.zhipu.assistant.AssistantType;
+import dev.langchain4j.community.model.zhipu.assistant.conversation.ConversationId;
+import dev.langchain4j.community.model.zhipu.assistant.conversation.ConversationRequest;
+import dev.langchain4j.community.model.zhipu.assistant.problem.Problems;
+import dev.langchain4j.community.model.zhipu.spi.ZhipuAssistantChatModelBuilderFactory;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.model.StreamingResponseHandler;
+import java.time.Duration;
+import java.util.List;
+
+public class ZhipuAssistantChatModel {
+
+    private final String appId;
+    private final ZhipuAssistantClient client;
+
+    public ZhipuAssistantChatModel(
+            String baseUrl,
+            String apiKey,
+            String appId,
+            Boolean logRequests,
+            Boolean logResponses,
+            Duration callTimeout,
+            Duration connectTimeout,
+            Duration readTimeout,
+            Duration writeTimeout) {
+        this.appId = appId;
+        this.client = ZhipuAssistantClient.builder()
+                .baseUrl(getOrDefault(baseUrl, "https://open.bigmodel.cn/"))
+                .apiKey(apiKey)
+                .callTimeout(callTimeout)
```

Comment:
```suggestion
                .callTimeout(getOrDefault(callTimeout, ofSeconds(60)))
```

nit: I think it's better to have default value for the unimportant parameters.

Could you please add default value for all `timeout` parameters in other models? (e.g. `ZhipuAiChatModel`, `ZhipuAiEmbeddingModel`, etc.)